_posts/<KJEZ35>/2020-09-02-KJEZ35-custom-turtle

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+author: KJEZ35
+title: "KJEZ's Custom Turtle"
+---
+
+KJEZ's Custom Turtle will go here 
+
+<iframe src="https://trinket.io/embed/python/de97b98fb8" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+![turtle image]<iframe src="https://trinket.io/embed/python/de97b98fb8" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
Nice experiment with markdown images, but this isn't quite how they work. I'm going to merge this so we can take a look at why.